docs/CNVs/CNV-methods.tex

```diff
@@ -829,44 +916,99 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
 
-\item $\vv_t = \sum_s \bar{\Psi}_{t}^{-1} (m_t - m_{st})E[\vz_s ]$ is $O(D T S)$.
+\subsubsection{GC bias correction}
+We can easily integrate sample-specific GC bias into this model.  Let $f_s({\rm GC})$ be the GC bias of GC content ${\rm GC}$ for sample $s$.  Then this enters into the model as an additional multiplier to the Poisson parameter.  That is, we replace $d_s c_{st} \rightarrow d_s c_{st} f_s({\rm GC}_t)$, which affects the model learning and inference only via the definition of $m_{st}$.  We can iteratively re-estimate the GC bias function $f_s$ by regressing the bias not explained by the latent factors.  That is, for each target the Poisson parameter is, ignoring GC effects,
+\begin{equation}
+c_{st} d_s \exp \left( \vW_t \EE[\vz_s] + m_t \right)
+\end{equation}
+and thus the ratio $n_{st} / \left[ c_{st} d_s \exp \left( \vW_t \EE[\vz_s] + m_t \right) \right]$ (with error bars of size $1/\sqrt{n}$ if we want to do a weighted regression) is an estimate of $f_s({\rm GC}_t)$ that we can feed into our favorite regression model.  This is more sophisticated than the standard approach of simply regressing $n$ versus ${\rm GC}$ in that it seeks to explain with GC only the bias that cannot be explained by linear latent features.
 
-\item $ \vW_{t \cdot} = (\vA + \vQ)^{-1} \vv_t$ is $O(D^2 T)$.
+\subsubsection{PCA and the curse of small samples}
+PCA-like denoising approaches minimize the {\em total variance} by learning and subtracting the contribution of the underlying latent features from the data. In practice, this objective is acheived using either the maximum variance principle (usual PCA) or the maximum likelihood principle on a linear-Gaussian model as explained earlier. In either method, when the number of samples largely exceeds the dimension of the latent space, sample-specific variations become immaterial and the true underlying latent features can be learned from the data. On the other hand, when the number of samples is comparable to the number of latent features, the statistical power for separating bsample-specific variations from mutual variations is significantly reduced.
 
-\item $\ave{W_{t\mu}^2} = W_{t\mu}^2 + (\vA + \vQ)^{-1}_{\mu\mu}$ is $O(D T)$.
+Let us assume that we have an oracle for the first $N$ major latent features, and that we have already subtracted the contribution arising from these features. Let $\sigma_\ell^2$ further be the variance associated to next leading latent feature such that subtracting it reduces the total variance by $N\sigma_\ell^2$. Now, if one of the samples has an individual leftover variance of magnitude $\sigma_S^2$ such that $\sigma_S^2 \gtrsim N\sigma_\ell^2$, then the maximum variance principle implies choosing the next principal component along the direction of that specific sample. In other words, the procedure erroneously learns a sample-specific signal as a source of noise. Note that this artifact occurs only if $N \lesssim \sigma_S^2 / \sigma_\ell^2$.\\
```

Comment:
Yes, and I think it would also be good to reiterate here the meaning of $S$.  As it reads now, it's only implied that $N$ is supposed to be the number of samples and that we just happen to know the same number of latent features.